docs/source/pipelines.rst

```diff
@@ -0,0 +1,541 @@
+            ratios for the |split step| of the :ref:`MLflow Regression Pipeline
+            <mlflow-regression-pipeline>`.  This configuration demonstrates use of
+            |Jinja2| templating syntax to define a dynamic value for the ``split_ratios``
+            attribute that can be overriden by
+            :ref:`Pipeline Profiles <profiles-key-concept>`.
+
+  steps:
+    split:
+      # Train/validation/test split ratios
+      split_ratios: {{SPLIT_RATIOS|default([0.75, 0.125, 0.125])}}
+
+.. _pipelines-key-concept:
+
+Pipelines
+~~~~~~~~~
+
+A Pipeline is a composition of :ref:`Steps <steps-key-concept>` used to solve an ML problem
+or perform an MLOps task, such as developing an accurate regression model for data within a
+particular domain or distributing a model to perform scalable batch inference. Pipelines
+structure their code, configurations, and tests as git repositories with a standardized
+directory hierarchy, as discussed in :ref:`Pipeline Repositories
+<pipeline-repositories-key-concept>`. MLflow Pipelines includes predefined pipeline repositories
+for common ML problems and MLOps tasks called :ref:`Pipeline Templates <pipeline-templates>`.
+MLflow Pipelines also provides :py:class:`APIs <mlflow.pipelines.Pipeline>` and a :ref:`CLI <cli>`
+for running pipelines and inspecting their results.
+
+.. code-section::
+
+    .. code-block:: python
+      :caption: Example API and CLI workflows for running the :ref:`MLflow Regression Pipeline
+                <mlflow-regression-pipeline>` and inspecting results. Note that pipelines
+                must be run from within their corresponding git repositories.
+
+      import os
+      from mlflow.pipelines import Pipeline
+      from mlflow.pyfunc import PyFuncModel
+
+      os.chdir("~/mlp-regression-template")
+      regression_pipeline = Pipeline(profile="local")
+      # Run the full pipeline
+      regression_pipeline.run()
+      # Inspect the model training results
+      regression_pipeline.inspect(step="train")
+      # Load the trained model
+      regression_model_pipeline: PyFuncModel = regression_pipeline.get_artifact("model")
+
+    .. code-block:: sh
+
+      git clone https://github.com/mlflow/mlp-regression-template
+      cd mlp-regression-template
+      # Run the full pipeline
+      mlflow pipelines run --profile local
+      # Inspect the model training results
+      mlflow pipelines inspect --step train --profile local
+      # Inspect the resulting model performance evaluations
+      mlflow pipelines inspect --step evaluate --profile local
+
+.. _pipeline-repositories-key-concept:
+
+Pipeline Repositories
+#####################
+
+Pipelines are structured as git repositories with a standardized directory hierarchy. The following
+example pipeline repository layout adapted from the |MLflow Regression Pipeline repository|
+showcases the main components that are common across all pipeline repositories.
```

Comment:
```suggestion
shows the main components that are common across all pipeline repositories.
```

Reply:
Or illustrates

Reply:
I think the issue here was less the word choice and more how long / wordy the sentence was. Broke this up / changed to

```
Pipelines are structured as git repositories with a standardized directory hierarchy. The following
example provides an overview of the pipeline repository structure.  It is adapted from the
|MLflow Regression Pipeline repository|.
```

Lmk what you think :)

Reply:
Amazing!